src/generation/generate.rs

```diff
@@ -7844,12 +7853,7 @@ fn gen_conditional_brace_body<'a>(opts: GenConditionalBraceBodyOptions<'a>, cont
   let end_statements_lc = LineAndColumn::new("endStatements");
   let header_trailing_comments = get_header_trailing_comments(opts.body_node, context);
   let body_should_be_multi_line = get_body_should_be_multi_line(opts.body_node, &header_trailing_comments, context);
-  let should_use_new_line = get_should_use_new_line(
```

Comment:
Oops, I forgot to format things on my PR, sorry! (ironic?)

Reply:
No, that's ok. I don't enforce formatting on the CI 😄